### PR TITLE
Move to navigate route by default

### PR DESCRIPTION
**Pull request by goshander**

### CODE REVIEW

File: src/ui/units/dash/modules/helpers.ts [hunk 1]
```diff
@@ -176,7 +176,7 @@ export function addOperationForValue(args: {
 
 export function getChartEditLink(entryId: string) {
     return Utils.isEnabledFeature(Feature.UseNavigation)
-        ? `${window.DL.endpoints.charts}/navigation/${entryId}`
+        ? `${window.DL.endpoints.charts}/navigate/${entryId}`
```
Comment (jhoncool) [inline src/ui/units/dash/modules/helpers.ts:179]: I think here should be `/navigate/${entryId}` or `${window.location.origin}/navigate/${entryId}`

Comment (goshander) [reply src/ui/units/dash/modules/helpers.ts]: Yep, I fix it a second ago 🙏

/navigation - for old behaviour with enabled UseNavigation flag
/navigate - for new route wuth support not only for wizard

Comment (jhoncool) [reply src/ui/units/dash/modules/helpers.ts]: Are you sure that we need window.DL.endpoints.charts for new route?

File: src/ui/units/dash/modules/helpers.ts [hunk 2]
```diff
@@ -177,7 +177,7 @@ export function addOperationForValue(args: {
 export function getChartEditLink(entryId: string) {
     return Utils.isEnabledFeature(Feature.UseNavigation)
         ? `${window.DL.endpoints.charts}/navigation/${entryId}`
-        : `${window.DL.endpoints.wizard}/${entryId}`;
+        : `${window.DL.endpoints.charts}/navigate/${entryId}`;
```
Comment (jhoncool) [inline src/ui/units/dash/modules/helpers.ts:180]:
```suggestion
        : `${window.location.origin}/navigate/${entryId}`;
```

**Comment by goshander** [comment]
@jhoncool change to simple `/navigate`

File: src/ui/units/dash/modules/helpers.ts (hunk 1 above)
Comment (goshander) [reply src/ui/units/dash/modules/helpers.ts]: No, you are right

File: src/ui/libs/DatalensChartkit/modules/menu/menu.ts [hunk 3]
```diff
@@ -58,9 +58,6 @@ export const getChartkitMenuByType = (props?: GetChartkitMenuByType) => {
             },
             [MenuItemsIds.EDIT]: {
                 title: i18n('dash.chartkit-menu.view', 'button_edit'),
-                action: ({propsData: {id, params}}) => {
```
Comment (jhoncool) [inline src/ui/libs/DatalensChartkit/modules/menu/menu.ts:61]: Why don't we need it?

Comment (goshander) [reply src/ui/libs/DatalensChartkit/modules/menu/menu.ts:61]: It is override default behaviour, but probably we can delete it later

**Review by jhoncool — APPROVED** [review]

**Pull request merged**